src/components/atoms/button/button.html

```diff
@@ -0,0 +1 @@
+<button class="btn-b1">btn-button</btn-text1></button>
```

Comment:
[課題ページ](https://shinonome.io/frontend-basic/lastAssignment/atoms_button/)にもありますが、aタグで作成しましょう。
また、閉じタグが残っているので修正をお願いします。
```suggestion
<a class="btn-b1">btn-button</a>
```